### ISSUE TITLE
Permission to use this repository configuration and content for Practicalli Neovim

### ISSUE THREAD

**Issue by practicalli-john** [rafaeldelboni/nvim-fennel-lsp-conjure-as-clojure-ide#13]
With your permission I'd like to use this configuration as the basis for the Neovim setup in the online book [Practicalli Neovim](https://practical.li/neovim/).  The book is still in very early stages, but would cover the Neovim setup in detail (base on your config with some other additions - which I can contribute back as PRs if desirable).

There is also a rough section on [using Neovim on an ultra-portable development environment](https://practical.li/neovim/termux/) (android smart phone with termux and Keyboard.io Atreus mechanical keyboard)

The book would also go into detail on using Neovim in general and for specific development tasks, mostly focused on Clojure.

I'd also like to add more plugins, such as which-key and a magit-like git client, as well as convienient key bindings to provide a similar level of features covered in the Practicalli Spacemacs book. 

I would naturally give full credit to all the work you have done and add any links you wish, such as a GitHub sponsors link.

All Practicalli on-line books are non-commercial and available under a creative commons license.

Please let me know if you have any concerns.

**Comment by rafaeldelboni** [comment]
Hi,

It would be an honor, I made this repository to get some clojurian friends into vim, so I decided to strip down my dotfiles and document how it works.

I was planning to add which-key and the labels for the custom commands in this configuration in a near future so a PR is super welcome.

About the links, I'm fine if you add my GitHub profile or [personal website](https://rafael.delboni.cc) :)

But to be honest the majority of the work/credit goes to the incredible @Olical and it's tools (Aniseed & Conjure), even this repo is heavily inspired in it's personal dotfiles.

_offtopic: I really like mechanical keyboards, planning to build an Atreus someday, I'm having a lot of fun with my [current one](https://github.com/rtitmuss/torn)_
